src/sbitx_gtk.c

```diff
@@ -2940,7 +2986,7 @@ void update_titlebar(){
 	time_t now = time_sbitx();
 	struct tm *tmp = gmtime(&now);
 //	sprintf(buff, "sBitx %s %s %04d/%02d/%02d %02d:%02d:%02dZ",  
-	sprintf(buff, "%s %s %s %04d/%02d/%02d %02d:%02d:%02dZ", 
+	sprintf(buff, "%s  %s  %s  %04d/%02d/%02d %0 2d:%02d:%02dZ", 
```

Comment:
Is this supposed to have a space around %0 2d?

Reply:
sprintf(buff, "%s %s %s %04d/%02d/%02d %02d:%02d:%02dZ",    seems correct to me...